src/content/glossary/index.md

```diff
@@ -530,6 +530,10 @@ A network [node](#node) that finds valid [proof-of-work](#pow) for new blocks, b
   Mining
 </DocLink>
 
+### Mint {#mint}
+
+Minting is the process of creating new tokens and bringing them into circulation so that they can be traded. It's a decentralized mechanism to create a new token without the involvement of the central authority.
```

Comment:
```suggestion
Minting is the process of creating new tokens and bringing them into circulation so that they can be used. It's a decentralized mechanism to create a new token without the involvement of the central authority.
```